.github/workflows/test-all.yml

```diff
@@ -2,11 +2,13 @@ name: "Test all packages"
 
 on:
   workflow_dispatch:
+    inputs:
+      which-gap:
```

Comment:
This section is where your PR and `main` have a conflict. Though it seems to me that this shouldn't be too hard to resolve?